classes/01class/exercises/c01-e03/drii-cavalcanti/c01-e03.txt

```diff
@@ -0,0 +1,32 @@
+Create a new EC2 Instance inside the default VPC with an public IP address and connect to it though SSH:
```

Comment:
this file needs to be removed from the PR.
Don't remove it from the repo, just from the PR...